syncer/serf/agent.go

```diff
@@ -124,3 +163,37 @@ func (a *Agent) UserEvent(name string, payload []byte, coalesce bool) error {
 func (a *Agent) Query(name string, payload []byte, params *serf.QueryParam) (*serf.QueryResponse, error) {
 	return a.Agent.Query(name, payload, params)
 }
+
+func (a *Agent) retryJoin(ctx context.Context) (err error) {
```

Comment:
Suggest to give some comments here to tell what is `retryJoin` 's repsonsiblity？